doc/sphinx/features.rst

```diff
@@ -16,126 +16,125 @@ header file:
 General features
 ----------------
 
--  By default, all coordinates in are periodic. With
-   ``PARTIAL_PERIODIC`` turned on, the global variable ``periodic`` (see
+-  ``PARTIAL_PERIODIC`` By default, all coordinates in |es| are periodic. With
+   ``PARTIAL_PERIODIC`` turned on, the |es| global variable ``periodic`` (see
    section ) controls the periodicity of the individual coordinates.
    Note that this slows the integrator down by around :math:`10-30\%`.
 
--  This switches on the various electrostatics algorithms, such as P3M.
+-  ``ELECTROSTATICS`` This switches on the various electrostatics algorithms, such as P3M.
```

Comment:
use "enables the use of" instead of "switches on". e.g. p3m still has to be tuned etc.

Reply:
If you are sure something should be changed, you can just do it right here. Click the pencil button on the top right.